pkgs/tools/system/btop/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "btop";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "aristocratos";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-pa9i65ndx8LMTMRXyL2GXCauM6Q8gAb16zGOylQFwL0=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    make install PREFIX=$out
+    runHook postInstall
+  '';
```

Comment:
```suggestion
  installFlags = [ "PREFIX=$out" ];
```

Reply:
For some reason this doesn't work. The result/ directory is empty when I build it this way.

Reply:
my bad, i forgot the parenthesis, edited the diff